packages/sirius-web/frontend/sirius-web-application/src/extension/DefaultExtensionRegistryMergeStrategy.ts

```diff
@@ -25,11 +25,15 @@ export class DefaultExtensionRegistryMergeStrategy implements ExtensionRegistryM
   ): ComponentExtension<any>[] {
     return [...existingValues, ...newValues];
   }
+
   public mergeDataExtensions(
     _identifier: string,
-    _existingValue: DataExtension<any>,
+    existingValue: DataExtension<any>,
     newValue: DataExtension<any>
   ): DataExtension<any> {
+    console.debug(
+      `The extension point with identifier ${existingValue.identifier} has been overwritten by ${newValue.identifier}`
```

Comment:
I'll update the message because technically it's an extension that is being overwritten not an extension point